src/main/java/nextstep/fp/Lambda.java

```diff
@@ -53,4 +57,8 @@ public static int sumAllOverThree(List<Integer> numbers) {
         }
         return total;
     }
+
+    public static int sum(List<Integer> numbers, Predicate<Integer> predicate) {
+        return numbers.stream().filter(predicate).reduce(0, Integer::sum);
```

Comment:
```suggestion
        return numbers.stream()
                .filter(predicate)
                .reduce(0, Integer::sum);
```
개인적으로는 이렇게 개행을 적절히 추가하는게 가독성이 좋더라구요 :)